### DISCUSSION TITLE
FSlab discord server

### DISCUSSION THREAD

**Discussion by kMutagene** [fslaborg/FSharp.Stats#151] [General]
Hi there! 

we are currently discussing on changing the Plotly.NET discord server to a FsLab server with channels for every project. 

The aim is to have a less fragmented community for FsLab, as well as provide a place for more in-depth questions for (possible) contributors. I was hesitant first as well, but doing that for Plotly.NET helped a lot getting a new contributor started.             

The only thing you would have to do is adding the link to the discord server to your readme eventually. 

Please let me know what you think about this.

Also, if you do not want to take actively part in this, please let me know if it is okay to create an 'unofficial' channel, where community members may try to help answering questions about your package.

**Comment by bvenn** [comment]
I like your suggestions and assume it makes the conversation faster and more efficient. Especially for new developers and users, this makes it easier to get started with fslab projects 👍
